packages/@aws-cdk/aws-codepipeline/lib/_enhanced_action.ts

```diff
@@ -0,0 +1,36 @@
+import { Aws, ResourceEnvironment } from '@aws-cdk/core';
```

Comment:
Isn't our standard to use `lib/private/` for private classes. What's the `_` prefix in the file name?

Reply:
We have both.

Reply:
Feels weird. We should standardize to one of the two.

Reply:
+1 on moving this to `private/` (there are a few different private files in this module already, maybe it makes sense to move those there as well).